ui/apps/platform/src/Containers/Vulnerabilities/utils/searchUtils.tsx

```diff
@@ -79,6 +77,22 @@ export function getEntityPagePath(
     }
 }
 
+export function getPlatformEntityPagePath(
```

Comment:
I'm an advocate of adding an appropriate amount of unit tests for deterministic functions like these, when you add the function.